android/app/src/test/java/co/nimblehq/smsforwarder/ui/screens/home/AllFiltersViewModelTest.kt

```diff
@@ -12,15 +13,15 @@ import org.amshove.kluent.mock
 import org.junit.Before
 import org.junit.Test
 
-class HomeViewModelTest {
+class AllFiltersViewModelTest {
```

Comment:
@lydiasama The package for the test classes still `home`?

Reply:
@hoangnguyen92dn Updated.